ssv/spectest/tests/runner/postconsensus/invalid_then_quorum.go

```diff
@@ -0,0 +1,183 @@
+package postconsensus
+
+import (
+	"fmt"
+
+	"github.com/attestantio/go-eth2-client/spec"
+
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// InvalidThenQuorum  tests a runner receiving an invalid message then a valid quorum, terminating successfully
+func InvalidThenQuorum() tests.SpecTest {
+	ks := testingutils.Testing4SharesSet()
+	expectedError := "failed processing post consensus message: invalid post-consensus message: could not verify Beacon partial Signature: wrong signature"
+	multiSpecTest := &tests.MultiMsgProcessingSpecTest{
+		Name: "post consensus invalid then quorum",
+		Tests: []*tests.MsgProcessingSpecTest{
+			{
+				Name: "sync committee contribution",
+				Runner: decideRunner(
```

Comment:
why decide runner?

Reply:
Because we first need to terminate the consensus to accept the post-consensus messages